scripts/bench.sh

```diff
@@ -0,0 +1,4 @@
+#! /usr/bin/env bash
+
+./bench-wasmer_sys.sh && \
```

Comment:
Any reason to `&&` rather than just sequential commands?

Reply:
I guess so that if the first command fails you see the failure output as the last text in the terminal rather than having it get buried by the output of the subsequent command. I don't have a strong preference either way.